lua/lint/linters/pytestcov.lua

```diff
@@ -0,0 +1,109 @@
+local messages = {
+  file_not_covered = 'No test coverage was found for this file',
+  multiple_lines_not_covered = 'No test coverage found for these lines',
+  line_not_covered = 'No test coverage found for this line',
+  testing_error = 'No test coverage analysis could be performed due to errors'
+}
+
+local severities = {
+  error = vim.diagnostic.severity.ERROR,
+  info = vim.diagnostic.severity.INFO,
+}
+
+local ignored_files = {
+  "setup.py",
+  "docs/conf.py",
+  "test_.*.py",
+  "__init__.py",
+}
+
+return {
+  cmd = 'pytest',
+  stdin = false,
+  args = {"--cov-reset", "--cov-report=term-missing", function()
+    local buffer_path = vim.fn.fnamemodify(vim.api.nvim_buf_get_name(bufnr), ":~:.")
+    return '--cov=' .. string.match(buffer_path, "(.*)/.*.py.*")
```

Comment:
This is just to retrieve the directory of the file, but maybe there are simpler ways